see also https://github.com/marbl/Mash/issues/167

Author: Andreas Tille <tille@debian.org>
Last-Update: Tue, 12 Oct 2021 09:01:12 +0200
Description: Fix "error: ‘numeric_limits’ is not a member of ‘std’"
Forwarded: https://github.com/marbl/Mash/issues/167

--- a/src/mash/robin_hood.h
+++ b/src/mash/robin_hood.h
@@ -45,6 +45,7 @@
 #include <memory> // only to support hash of smart pointers
 #include <stdexcept>
 #include <string>
+#include <limits>
 #include <type_traits>
 #include <utility>
 #if __cplusplus >= 201703L

